test/integration/build/aws.tf

```diff
@@ -1899,4 +1899,9 @@ resource "aws_vpc_endpoint" "vpc_endpoint_1" {
   tags = {
     Name = var.aws_vpc_endpoint_name
   }
-}
\ No newline at end of file
+}
+
+resource "aws_guardduty_detector" "detector_1" {
+  count  = 1
```

Comment:
can we make this `count = var.aws_enable_creation`

Reply:
Updated in latest commit.